csharp/src/AddressBook/AddressBook.csproj

```diff
@@ -41,6 +41,20 @@
     <GenerateSerializationAssemblies>Off</GenerateSerializationAssemblies>
     <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'ReleaseSigned|AnyCPU' ">

```

Comment:
I don't think we need a signed version of the tutorial. We can make the release solution configuration just use the release project configuration for this project.